unlock-app/src/components/interface/Layout.js

```diff
@@ -11,14 +11,17 @@ import { MessageBox } from './modal-templates/styles'
 import { ActionButton } from './buttons/ActionButton'
 import withConfig from '../../utils/withConfig'
 import useTermsOfService from '../../hooks/useTermsOfService'
+import { useProvider } from '../../hooks/useProvider'
 import Loading from './Loading'
 import GlobalErrorConsumer from './GlobalErrorConsumer'
 
 export default function Layout({ forContent, title, children }) {
   const { termsAccepted, saveTermsAccepted, termsLoading } = useTermsOfService()
-  if (termsLoading) {
+  const { loading: providerLoading } = useProvider()
```

Comment:
This will fix #6261 because we won't show anything until we actually have a provider available for use, or the certainty that we do not have any!